src/Components/Clash/ClashBox/ClashBox.css

```diff
@@ -2,4 +2,11 @@
     display: flex;
     margin: 4% 0;
     justify-content: space-between;
+}
+@media screen and (max-width:600px){
```

Comment:
Can I suggest making this media query come into effect at 1150px instead? There is a weird point just over 600px where things are a bit squashed, and setting it to that value seems to cover all screen sizes quite nicely!